assests/styling/QaStyling.css

```diff
@@ -0,0 +1,62 @@
+body{
+    font-size: 1.5rem;
+}
+
+#page-content {
+    width: 100%;
+    padding-right: 5rem;
+}
+
+#wrapper{
+    display: flex;
+    flex-direction: row;
+    /* border: 1px solid black; */
+    height: 50%;
+}
+#FirstQaDiv{
+    border: 1px solid black;
+    width: 20%;
+}
+#SecondQaDiv{
+    width:80%;
+}
+#Icon {
+color: dimgray;
+}
+
+
+tr:hover {
+    background-color: #73b99b;
+}
+#CategoryAdd{
+    float:right;
+}
+th{
```

Comment:
Pass op met het stylen van html tags. Kan zorgen voor kopzorgen later in het project. Probeer zo veel mogelijk met classes en id's te werken.